src/player.cpp

```diff
@@ -7684,7 +7684,7 @@ void player::suffer()
 
     if(has_active_mutation("WINGS_INSECT")){
         //~Sound of buzzing Insect Wings
-        sounds::sound( pos(), 10, "BZZZZZ");
+        sounds::sound( pos(), 10, _("BZZZZZ"));
```

Comment:
@narc0tiq Aye, good to know. The edit about plugging it into `sounds::sound()` though...